analysis/port_analyzer.go

```diff
@@ -63,50 +82,50 @@ func (h *PortAnalyzer) summarize() {
 		}
 	}
 
-	if h.inTrafficMsg > 0 {
-		h.PerfLogger.AddDataEntry(PerfAnalyzerEntry{
-			Start: startTime,
-			End:   endTime,
-			Where: h.port.Name(),
-			What:  "IncomingByte",
-			Value: float64(h.inTrafficByte),
-			Unit:  "Byte",
-		})
-
-		h.PerfLogger.AddDataEntry(PerfAnalyzerEntry{
-			Start: startTime,
-			End:   endTime,
-			Where: h.port.Name(),
-			What:  "IncomingMsg",
-			Value: float64(h.inTrafficMsg),
-			Unit:  "Msg",
-		})
-	}
-
-	if h.outTrafficMsg > 0 {
-		h.PerfLogger.AddDataEntry(PerfAnalyzerEntry{
-			Start: startTime,
-			End:   endTime,
-			Where: h.port.Name(),
-			What:  "OutGoingByte",
-			Value: float64(h.outTrafficByte),
-			Unit:  "Byte",
-		})
-
-		h.PerfLogger.AddDataEntry(PerfAnalyzerEntry{
-			Start: startTime,
-			End:   endTime,
-			Where: h.port.Name(),
-			What:  "OutGoingMsg",
-			Value: float64(h.outTrafficMsg),
-			Unit:  "Msg",
-		})
+	for dst, entry := range h.PortAnalyzerTable {
+		if entry.InTrafficMsg != 0 {
+			h.PerfLogger.AddDataEntry(PerfAnalyzerEntry{
```

Comment:
Let's slightly modify the `PerfAnalyzerEntry` to make it more straightfoward. Do not use `What` field to hold port name.